ibtk/include/ibtk/muParserRobinBcCoefs.h

```diff
@@ -194,28 +194,28 @@ class muParserRobinBcCoefs : public SAMRAI::solv::RobinBcCoefStrategy<NDIM>
     /*!
      * User-provided constants specified in the input file.
      */
-    std::map<std::string, double> d_constants;
+    std::map<std::string, double> d_constants = {};
 
     /*!
      * The strings providing the data-setting functions which are evaluated by the
      * mu::Parser objects.
      */
-    std::vector<std::string> d_acoef_function_strings;
-    std::vector<std::string> d_bcoef_function_strings;
-    std::vector<std::string> d_gcoef_function_strings;
+    std::vector<std::string> d_acoef_function_strings = {};
+    std::vector<std::string> d_bcoef_function_strings = {};
+    std::vector<std::string> d_gcoef_function_strings = {};
 
     /*!
      * The mu::Parser objects which evaluate the data-setting functions.
      */
-    std::vector<mu::Parser> d_acoef_parsers;
-    std::vector<mu::Parser> d_bcoef_parsers;
-    std::vector<mu::Parser> d_gcoef_parsers;
+    std::vector<mu::Parser> d_acoef_parsers = std::vector<mu::Parser>(2 * NDIM);
```

Comment:
Does `auto` work here?

Reply:
This is a strange declaration: it should suffice to just write
```cpp
std::vector<mu::Parser> d_acoef_parsers(2 * NDIM);
```

Reply:
@drwells I initialized like this to match the other initializations I did using `= <value>`. I can go back and update these kinds of initializations though to make them cleaner.

Reply:
Oh, I didn't realize that this was in a header. We might be stuck with this.